README.md

```diff
@@ -84,5 +84,18 @@ python manage.py migrate
 python manage.py runserver
 ```
 
+### Run the Test
+
+For running frontend tests, simply run
+```
+    yarn test
+```
+For running backend tests, run
+```
+    py.test backend
```

Comment:
according to this py.test seems to be the old version and that we should use pytest instead? https://stackoverflow.com/questions/39495429/py-test-vs-pytest-command

Are we using the wrong version of pytest or something?

Reply:
When I did `pytest`, it complaints about unrecognized arguments, in which arguments are from the other two plugins. But if I did `py.test`, it works fine. Not sure why